Frameworks/CoreFoundation/Base.subproj/CFRuntime.c

```diff
@@ -275,6 +275,10 @@ void _CFRuntimeBridgeTypeToClass(CFTypeID cf_typeID, const void *cls_ref) {
     __CFUnlock(&__CFBigRuntimeFunnel);
 }
 
+uintptr_t __CFISAForTypeID(CFTypeID typeID) {
```

Comment:
Like: in CT, CF_INLINE works fine? Even though this refers to __CFRuntimeObjCClassTable?

---
In reply to: [78866010](https://github.com/Microsoft/WinObjC/pull/1008#discussion_r78866010) [](ancestors = 78866010,78862337)